core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller.rb

```diff
@@ -97,7 +97,11 @@ def update
   # @response [object<errors: object<title: string>>] 404 Not found
   # @response 401 Not authorized
   def deploy_containers
-    deployment = resource_project.deployments.active.find(params[:id])
+    deployment = resource_project.deployments.existed.find(params[:id])
+
+    unless deployment.active?
+      render json: { errors: { title: ["Deployment #{deployment.state}"] } }, status: :unprocessable_entity
+    end
```

Comment:
I think we didn't describe the bug clearly enough.

In this version code when we start update the previous failed preview we immediately see the error `Deployment failed`. It's not good from a user's point of view
Therefore we should not render this error here. We should just start update deployment.

There is one place where raise `Not Found` error
`core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments/activity_items_controller.rb:18`

This action called by our CLI in cycle for check statuses. When our deployment gets status 'failed` this action raise `Not Found Deployment` error